packages/shared/src/features/onboarding/shared/StepHeadline.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import classNames from 'classnames';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../../components/typography/Typography';
+import { FunnelStepType } from '../types/funnel';
+
+export type StepHeadlineProps = {
+  headline: string;
+  explainer: string;
+  align?: 'left' | 'center';
+  visualUrl?: string;
+  type?: FunnelStepType;
+};
+
+const StepHeadline = ({
+  headline,
+  explainer,
+  type,
+  align = 'center',
+}: StepHeadlineProps): ReactElement => {
+  return (
+    <div
+      data-testid="step-headline-container"
+      className={classNames(
+        'flex flex-col gap-2',
+        align === 'center' ? 'text-center' : 'text-left',
+      )}
+    >
+      <Typography bold type={TypographyType.Title1}>
+        {headline}
+      </Typography>
+      <Typography
+        data-testid="step-headline-explainer"
+        type={TypographyType.Body}
+        color={
+          type === FunnelStepType.Quiz
+            ? TypographyColor.Tertiary
+            : TypographyColor.Primary
+        }
```

Comment:
Wouldn't it be more dynamic to simply accept color from the outside?

Reply:
I'm more into making this a simpler prop, we will just pass the color from the step layer. 
Editor may not know there is a prop for this. We will just silently pass from the quiz step!